docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Expo Updates
+sidebar_title: Expo Updates
+---
+
+version 0
+
+---
+
+## Introduction
+
+This is the specification for Expo Updates, a protocol for delivering updates to apps running on multiple platforms.
```

Comment:
More of a general question: Is Expo Updates specific to react-native? If so, it may be good to specify what an "update" is here. If not, we'll want to ensure this document is free from react-native terminology (bundler, JavaScript, etc).